test-utils/src/main/java/apoc/util/TestContainerUtil.java

```diff
@@ -183,10 +183,6 @@ public static void executeGradleTasks(File baseDir, String... tasks) {
         }
     }
 
-    public static void executeGradleTasks(String... tasks) {
-        executeGradleTasks(baseDir, tasks);
```

Comment:
There are a few commented out dead lines of code in this file that might be nice to remove as well